app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -465,7 +467,8 @@ private void handleItemEvents(final ChatViewHolder chatViewHolder, final int pos
                         break;
                     case SUSI_MESSAGE:
                         chatViewHolder.messageStar.setVisibility( (model.isImportant()) ? View.VISIBLE : View.GONE);
-                        chatViewHolder.chatTextView.setText(model.getContent());
+                        chatViewHolder.chatTextView.setText(Html.fromHtml(model.getContent()));
```

Comment:
Ni need of this line. It is already parsed [here](https://github.com/fossasia/susi_android/blob/development/app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java#L1085)